packages/keys/src/key-pair.ts

```diff
@@ -41,3 +42,19 @@ export async function createKeyPairFromBytes(bytes: ReadonlyUint8Array, extracta
 
     return { privateKey, publicKey } as CryptoKeyPair;
 }
+
+export async function createKeyPairFromPrivateKeyBytes(
+    bytes: ReadonlyUint8Array,
+    extractable: boolean = false,
+): Promise<CryptoKeyPair> {
+    const promises = [createPrivateKeyFromBytes(bytes, true)];
+
+    if (!extractable) {
+        promises.push(createPrivateKeyFromBytes(bytes, false));
+    }
```

Comment:
I think it'd be worth adding a comment to explain why we do this

Reply:
I've added some comments to explain all that. 👌